docs/javascript/files.md

```diff
@@ -1,3 +1,7 @@
+---
+keywords: file, fileattachment, attachment
```

Comment:
just earlier today i was trying to get to this page typing blindly fast and landed on file input!! thank you 🙏